x-pack/plugins/security_solution/public/entity_analytics/components/risk_information/index.tsx

```diff
@@ -33,10 +32,20 @@ import * as i18n from './translations';
 import { useOnOpenCloseHandler } from '../../../helper_hooks';
 import { RiskScoreLevel } from '../severity/common';
 import { RiskScoreEntity, RiskSeverity } from '../../../../common/search_strategy';
+import type { CriticalityLevel } from '../../../../common/entity_analytics/asset_criticality/types';
+import { CriticalityModifiers } from '../../../../common/entity_analytics/asset_criticality';
 import { RiskScoreDocLink } from '../risk_score_onboarding/risk_score_doc_link';
 import { BETA } from '../risk_score_onboarding/translations';
+import { AssetCriticalityBadge } from '../asset_criticality';
 
-const getTableColumns = (riskEntity?: RiskScoreEntity): Array<EuiBasicTableColumn<TableItem>> => [
+interface RiskLevelTableItem {
+  range?: string;
+  level: RiskSeverity;
+}
+
+const getRiskLevelTableColumns = (
+  riskEntity?: RiskScoreEntity
```

Comment:
@tiansivive it doesn't look like we ever pass `riskEntity` into this function, which means that the subsequent `INFORMATION_RISK_HEADER` function also doesn't need to accept an argument/be a function. I left it here for now, but I can clean that up real quick if you agree.

Reply:
yeah, it seems like it's not doing anything since we don't really care about making a distinction between different entities here. Let's clean it up.

Reply:
abd85b1ce09be60a415b5221999c11222bd8ee33